packages/shared/tailwind.config.js

```diff
@@ -183,6 +183,7 @@ module.exports = {
       desktopL: '2156px',
       mouse: { raw: '(pointer: fine)' },
       responsiveModalBreakpoint: '420px',
+      laptopH: { raw: '(min-height: 800px)' },
```

Comment:
Just wanna make sure this is part of our design system? cc: @rebelchris

Reply:
Since I moved to a css module instead, could probably remove this one.

Reply:
I think one or the other is better then doing it in two separate places. 👍

Reply:
Think the other place is better. The video is kinda difficult to tweak with just tailwind

Reply:
Yeah this one we can remove, it was the better approach when all we needed was one media query
I think this whole video scaling got a bit out of scope, but it's done now.

Reply:
Removed this one.

Reply:
Re-added. Forgot I had already used it two other places to tweak the styles on short screens

Reply:
I would prefer if we can stick to not using custom measurements for elements that don't need it.
Is it really only on height this one needs to add the padding?

I hoped we could simply use the existing `tablet:` or something for this part.

Reply:
I could also just throw it into the css file since they are both connected